go-controller/hack/update-codegen.sh

```diff
@@ -113,7 +104,5 @@ echo "Copying adminpolicybasedexternalroutes CRD"
 cp _output/crds/k8s.ovn.org_adminpolicybasedexternalroutes.yaml ../dist/templates/k8s.ovn.org_adminpolicybasedexternalroutes.yaml.j2
 echo "Copying egressService CRD"
 cp _output/crds/k8s.ovn.org_egressservices.yaml ../dist/templates/k8s.ovn.org_egressservices.yaml.j2
-echo "Copying IPAMClaim CRD"
```

Comment:
@flavio-fernandes this is what I had in mind, I think with Jaime's PR we can close your PR

Reply:
oh, ack! TY @tssurya

Reply:
Closing https://github.com/ovn-org/ovn-kubernetes/pull/4748